crategen/models/wes_models.py

```diff
@@ -0,0 +1,229 @@
+"""
+Each model in this module conforms to the corresponding WES model names as specified by the GA4GH schema (https://ga4gh.github.io/workflow-execution-service-schemas/docs/).
+
+This module provides Pydantic models for the Workflow Execution Service (WES) schema,
+supporting validation, serialization, and deserialization of WES data structures.
+"""
+from enum import Enum
+from typing import List, Optional, Union
+
+from pydantic import BaseModel, Field, root_validator, validator
+from rfc3339_validator import validate_rfc3339  # type: ignore
+
+# Define the maximum number of system logs allowed
+MAX_SYSTEM_LOGS = 2
+
+
+class WESState(str, Enum):
```

Comment:
for the class names i think its best to just use the exact names listed in the WES docs. This would ensure that the references match exactly and make all this much simpler to read.